doc/tutorials/stream_events.md

```diff
@@ -0,0 +1,297 @@
+# Tutorial: Listening to stream events #########################################
+
+Some contents contain events a player will need to send at a particular point
+in time. We call those in the RxPlayer "stream events".
+
+For example, stream events are often used jointly with ad-insertion, to allow a
+player to notify when an user begin to see a particular ad.
+
+Stream events are not only restrained to ad-related usages though. Any event
+you want to synchronize with the played content can be inserted.
+
+
+
+## Event Formats understood by the RxPlayer ####################################
+
+### DASH EventStream elements ##################################################
+
+For now, the RxPlayer only make use of DASH' EventStream elements.
+
+Such elements are defined in a DASH MPD in the concerned `Period`.
+Here is an example of such element in an MPD:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<MPD
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:mpeg:dash:schema:mpd:2011"
+  xsi:schemaLocation="urn:mpeg:dash:schema:mpd:2011 DASH-MPD.xsd"
+  type="dynamic"
+  minimumUpdatePeriod="PT2S"
+  timeShiftBufferDepth="PT30M"
+  availabilityStartTime="2011-12-25T12:30:00"
+  minBufferTime="PT4S"
+  profiles="urn:mpeg:dash:profile:isoff-live:2011">
+
+    <Period id="1">
+      <EventStream schemeIdUri="urn:uuid:XYZY" timescale="1000" value="call">
+        <Event presentationTime="0" duration="10000" id="0">
+          1 800 10101010
+        </Event>
+        <Event presentationTime="20000" duration="10000" id="1">
+          1 800 10101011
+        </Event>
+        <Event presentationTime="40000" duration="10000" id="2">
+          1 800 10101012
+        </Event>
+        <Event presentationTime="60000" duration="10000" id="3">
+          1 800 10101013
+        </Event>
+      </EventStream>
+      <!-- ... -->
+    </Period>
+
+</MPD>
+```
+
+Here the `<EventStream />` elements and its `<Event />` children elements will
+be parsed by the RxPlayer.
+
+Each `<Event />` element can then be sent through a single RxPlayer events.
+
+
+
+## How to listen to stream events ##############################################
+
+The RxPlayer notify of such events through the usual RxPlayer events.
+
+_As a reminder (or if you didn't know), the RxPlayer can send a multitude of
+[events](../api/player_events.md) that can be listened to by the usage of the
+[addEventListener method](../api/index.md#meth-addEventListener)._
+
+The events related to stream events are:
+
+  - `"streamEvent"`: an event has just been reached.
+
+  - `"streamEventSkipped"`: an event has been skipped over. This usually means
```

Comment:
So know it is officialy `streamEventSkip`

Reply:
Yep did not take the time yet to update! (See comment just before)

Reply:
now* 😛

Reply:
Done